warwomb/data_files/warcaster_units.txt

```diff
@@ -0,0 +1,643 @@
+[
+    {
+        "factions": ["AC", "EMP", "MW", "ISA"],
+        "type": "SOLO",

```

Comment:
I'd prefer title-case for the types, and no underscores for qualities ("Wild Card"), please

Reply:
See comment below.